skins/oasis/modules/templates/Body_Index.php

```diff
@@ -7,6 +7,14 @@
 /** @var $isEditPage bool */
 ?>
 
+<? if ( !empty( $wg->InContextTranslationsProject ) ): ?>
+	<script type="text/javascript">
+		var _jipt = [];
+		_jipt.push(['project', '<?= $wg->InContextTranslationsProject ?>' ]);
```

Comment:
Two things here. First - what's wrong with changing it to one liner, like ```var _jipt = [['project', 'xxx']];```. Second, just to be safe, you should use call something like `addslashes` before pasting a PHP variable into a JS string.